test/projectile-test.el

```diff
@@ -66,3 +85,29 @@ You'd normally combine this with `projectile-test-with-sandbox'."
     (expect (projectile-expand-root "foo") :to-equal "/path/to/project/foo")
     (expect (projectile-expand-root "foo/bar") :to-equal "/path/to/project/foo/bar")
     (expect (projectile-expand-root "./foo/bar") :to-equal "/path/to/project/foo/bar")))
+
+(describe "projectile-save-known-projects"
+  (it "saves known projects through serialization functions"
+    (projectile-mock-serialization-functions
```

Comment:
Just use `spy-on` here for the two functions and you can get rid of that ugly list quoting as well.